### DISCUSSION TITLE
能否设置直链下载？而不是显示一个页面才能下载

### DISCUSSION THREAD

**Discussion by yutianhot** [AlistGo/alist#565] [Q&A] [answered]
https://download.filezilla.cn/client/windows/FileZilla_3.57.0_win32-setup.exe

比如，这样打开是一个页面，要在页面点下载按钮才行。

没找到设置的方法

**Comment by xhofe** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/36558727/154785967-4f5b224e-db2c-441f-aeee-47a7738cfe92.png)

**Reply by yutianhot** [reply]
大佬，有办法去掉链接的D吗？

https://download.filezilla.cn/d/client/windows/FileZilla_3.57.0_win64.zip

**Reply by xhofe** [reply]
没有
